### DISCUSSION TITLE
Implementing composeView.getMessageId()

### DISCUSSION THREAD

**Discussion by designcode** [InboxSDK/InboxSDK#1053] [Ideas]
Hi

There is a method available on ComposeView called `getThreadID`, I was wondering if there is a way to find out `messageId` to which you're replying?

**Comment by wegry** [comment]
@designcode Going off the at the title of the discussion: we may already have a method that does the trick: https://github.com/InboxSDK/InboxSDK/commit/765c689608e99a4f15f38ef14e9ef515994af6dc#diff-ba1e4df2ce488e4d02c9be525b07542ce17c1543d83a850b0a91eca9a511109cR118

If this method does the trick for you, we could potentially make this method part of the public API.

**Reply by designcode** [reply]
@wegry I think this is slightly different then what I had in mind. I think this is already part of the public API. I was wondering if we can get the `gmailMessageId` of the message we are replying to. Something equivalent of `messageView.getMessageIDAsync()`
